test/framework/resources/prometheus-web-tls-secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  tls.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUZUekNDQXplZ0F3SUJBZ0lVUVNNNVBRbllxVDBoaDlhV3prOUNVdS9sUUNrd0RRWUpLb1pJaHZjTkFRRUwKQlFBd056RUxNQWtHQTFVRUJoTUNSRVV4RXpBUkJnTlZCQW9NQ2xCeWIyMWxkR2hsZFhNeEV6QVJCZ05WQkFNTQpDbEJ5YjIxbGRHaGxkWE13SGhjTk1qRXdOVEExTURjeU5UQTBXaGNOTWpJd05UQTFNRGN5TlRBMFdqQTNNUXN3CkNRWURWUVFHRXdKRVJURVRNQkVHQTFVRUNnd0tVSEp2YldWMGFHVjFjekVUTUJFR0ExVUVBd3dLVUhKdmJXVjAKYUdWMWN6Q0NBaUl3RFFZSktvWklodmNOQVFFQkJRQURnZ0lQQURDQ0Fnb0NnZ0lCQU9yUlg4NmluQ2ZzcUxXQQpqM2xDd3YxRmJKSG15TTNFK25kVVhLelRMUytwa21oL1phMlVPQmdZZ00raVYrNnBEeG0zZEk0OXVWNnc5MFBPCkxyNFJVY25mOWxRT2NDeWE4ZWFiSnJhUWNramFzOWhGZ0txZ1BVN24rdld3MjVyc25OUGtIeDNjQWdvajZHZDcKMmJlczJ5YkZVc0J0VWROVDVudGptbm5UQ0VJNVF6dUEyNHAvTnBod1BlT2VaS2NLYU96TXNKQ08zWUxsSVg2SAo3OTBSOGdoK2pyOGNmWWQ1RmN5RUxyaFhTazFub2gwa1gwelpTNlA5NzcrTVRmeEgrZ0VwTTA4RmM0SHVJdHZ5CkFQeXEvWE1mblpJUFdjbnhRanNDTEsvSERMa2dVMG9iV2RJdnNnS3FQOUJCQ0JPV3l1MGdpeXp4WkdlWGpoUU4KU1ltN3Y2YUZRVlVNSjNlMHUvWkwwVVpNSWw5Q2dHRW1uOXFWWUpNUG5ZWkxSZHgzd21BNmh4QnFWUGxiZkFoVQp0TFhPVTBRMncwMDVyd08vdWZKcUgzOXY5ZzYzeHY1dGdGbTh6ejVKOVNxMUQvMy9ZY3htdXVjNEllQUt1d3BsCi9QWHZmS3IzZEJZUVp0ZnJna3JWdXdaUTFSNGdNeUttcDF3SDdaVm5lWi9ibGpyVGtPZUFObXp5U0N0Z2lsZU4KTm10Q0JaZDg1UjAvWUtYNWs2WmJBcnZUK2tSZzhiY3lOZk5PRnVKb0J1a2RLN3UrM3JpQTVQL2EwRkh6c29RcApEUnd1bDdJN1ZvOExHaG5JbXFEWnNKcTBSeWhyWmpPNTFiaUhGQ3U0QWJPY1dBamRKVmhWVjZSclBBeXM5MGsxCk9GNGd6bEpYdG1lSnVMRU81TGlINVdUdjlnMWxBZ01CQUFHalV6QlJNRThHQTFVZEVRUklNRWFDQ25CeWIyMWwKZEdobGRYT0NFbkJ5YjIxbGRHaGxkWE11WkdWbVlYVnNkSUlrY0hKdmJXVjBhR1YxY3k1a1pXWmhkV3gwTG5OMgpZeTVqYkhWemRHVnlMbXh2WTJGc01BMEdDU3FHU0liM0RRRUJDd1VBQTRJQ0FRQ29jSFh6VzVxSU9PVDJJY1gwCnJLaGpvS0hlTWFVOUR3bW1MdG80RTQ4VHlrazhsQ1p3TjhkUXFhYXA4TFczSFdReGZvWWJDSHlSTkRxZCtRdmoKenNCdUZhcUJJZ0ZHcDlsbERhNWtLYlN2cXpSZmdTWWVPVG5keTBEdFlMYTVOU1p4d05VVmRmSXRVY2o2MW1vWgprd01odWhPa21kc3E1R2NKRng4dmhwUWdZN3M2UDVmV3RhbkwwZEZLNEVRbWN6OHpoNjh6TFB0dTYzWStWL1o1CjBhZ0FEL09iazdITUxkSjZjQlNCczJXcnpNZm5CaENHWWVQb3ZneGs2ejA1bThSeU84WFN2S1VwVHhHd0JSUmgKN0dxSXdEWmMvWjNNdEFjUTAyUXRoVzBiQlhwVHQrcS9PaXZOdHBXK0pMT0wrSUVGSXBBKzllWUpWbCt4NnFCMwo2aVRYc3l5aG5waGdFKzR5V0FuODdKcUZ3Ly9ZL1dmbk9oU1hVZFBFZThrZDl3VDNqRzdvd3AvSFNLOFcwU04wCmdRODNQelVZSHU3bDVrOEFIR1llNmd4WkpoUUZDTkVTWXlOalV5Ulk1Mk5oU2QrdENRcHFqdnA3T2kvZjBWWTcKTzl4WjlKeUord2F6Zk03UFl3MXAzNldWcnhqRDYzdGlnVjc5YmVXbXFZZTZuenpHYkJzN3hQaW02djJVdmd5ZApEL2I5dmIzSUVLbDd5UlQvamlOV2RET2RyczVJenVNYjd1T2NXY21FM2RmeXhvWU9tUG1QL2oxdmtCQ3lCdlhMCm0vMlRGclZJV1RPNUVES0EybmY0bmdTakNmUy8vSnQvdk1Ld002dEt0VER5ekFIcWFpYWtyTmdWNkphemNBSi8KcHpMbVFGMTNHU1J4Snd2Z000NGUyUnFlbHc9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
This certificate should have a longer validity period otherwise tests will start to fail after `May  5 07:25:04 2022 GMT`. Let's increase it to 10 years instead of a year.